pytravharv/SubjPropPathAssertion.py

```diff
@@ -120,13 +110,21 @@ def _assert_at_depth(self):
             "Asserting a property path for a given subject at a given depth"
         )
         log.debug("Depth: {}".format(self.max_depth - self.current_depth))
-        SPARQLQuery = self._sparql_trajectory_check(
-            self.max_depth - self.current_depth
-        )
-        if self.target_store.verify(SPARQLQuery):
+        if self.rdf_store_access.verify(
+            self.subject,
+            self.assertion_path.get_path_for_depth(
+                self.max_depth - self.current_depth
+            ),
+            self.prefix_set,
+        ):
             self._harvest_and_surface()
             return
-        self.target_store.ingest(WebAccess(self.subject), self.config_filename)
+        print(f"adding {self.subject} to {self.graph_name}")
+        graph_to_ingest = web_access(self.subject)
+        print(len(graph_to_ingest))
+        self.rdf_store_access.ingest(
+            web_access(self.subject), graph_name_to_uri(self.graph_name)
```

Comment:
why call `web_access` again?

also -- strange name for 2nd argument? confuses me -- what is this graph_name if not a uri to a named_graph already?